best_practices.rst

```diff
@@ -299,6 +299,10 @@ to add buttons in the templates. This also improves the separation of concerns,
 because the button styling (CSS class and other attributes) is defined in the
 template instead of in a PHP class.
 
+However, if you create a :doc:`form with multiple submit forms </form/multiple_buttons>`
+you should define them in the form class instead of the template. Otherwise, you
```

Comment:
I would actually rather recommend to add them in the controller. That's the place where you know that you want to check for which button was clicked and you don't prevent your form type from being embedded somewhere else.